app/code/Magento/QuoteGraphQl/Model/Resolver/ShippingAddress/AvailableShippingMethods.php

```diff
@@ -65,13 +66,19 @@ public function resolve(Field $field, $context, ResolveInfo $info, array $value
         $shippingRates = $address->getGroupedAllShippingRates();
         foreach ($shippingRates as $carrierRates) {
             foreach ($carrierRates as $rate) {
-                $methods[] = $this->dataObjectConverter->toFlatArray(
+                $method = $this->dataObjectConverter->toFlatArray(
                     $this->shippingMethodConverter->modelToDataObject($rate, $cart->getQuoteCurrencyCode()),
                     [],
                     ShippingMethodInterface::class
                 );
+                if ($method['available'] && $method['error_message'] === "") {
+                    $methods[] = $method;
+                }
             }
         }
+        if (count($methods) === 0) {
+            throw new GraphQlNoSuchEntityException(__(' This shipping method is not available. To use this shipping method, please contact us.'));
```

Comment:
Is it possible to load the text of the message from config [there](
https://github.com/magento/graphql-ce/blob/7b885666e9115441fe136202c60dc41044311b2f/app/code/Magento/OfflineShipping/Model/Carrier/Freeshipping.php#L104)

Have no idea how it can be done. So it's a question, not suggestion.